packages/user-group-service/src/api-keys/typedef.graphql

```diff
@@ -41,7 +41,7 @@ type APIAccess {
   microservice: String
 }
 
-union APIKeyOwner = UserType | Group
+union APIKeyOwner = RoverUserType | Group
```

Comment:
Can you undo this. As it might break the existing working of the API Key endpoints.